includes/class-wc-payments-utils.php

```diff
@@ -338,26 +338,39 @@ public static function map_search_orders_to_charge_ids( $search ) {
 	}
 
 	/**
-	 * Extract the billing details from the WC order
+	 * Extract the billing details from the WC order.
+	 * It only returns the fields that are present in the billing section of the checkout.
 	 *
 	 * @param WC_Order $order Order to extract the billing details from.
 	 *
 	 * @return array
 	 */
 	public static function get_billing_details_from_order( $order ) {
-		$billing_details = [
-			'address' => [
-				'city'        => $order->get_billing_city(),
-				'country'     => $order->get_billing_country(),
-				'line1'       => $order->get_billing_address_1(),
-				'line2'       => $order->get_billing_address_2(),
-				'postal_code' => $order->get_billing_postcode(),
-				'state'       => $order->get_billing_state(),
-			],
-			'email'   => $order->get_billing_email(),
-			'name'    => trim( $order->get_formatted_billing_full_name() ),
-			'phone'   => $order->get_billing_phone(),
+		$billing_fields       = array_keys( wc()->checkout()->get_checkout_fields( 'billing' ) );
```

Comment:
```suggestion
		$billing_fields       = array_keys( WC()->checkout()->get_checkout_fields( 'billing' ) );
```
I know that PHP will treat them the same, but thoughts about using the capitalised version of this global function, as it's named in WooCommerce, just for consistency with other places in the codebase where this function is used?